java.security-openjdk-7

```diff
@@ -0,0 +1,900 @@
+#
```

Comment:
This java.security is the default for required a custom order of `security.providers`, otherwise is pristine. I could just `perl/sed` the default file.

```
security.provider.1=sun.security.provider.Sun
security.provider.2=sun.security.rsa.SunRsaSign
security.provider.3=sun.security.ec.SunEC
security.provider.4=com.sun.net.ssl.internal.ssl.Provider
security.provider.5=com.sun.crypto.provider.SunJCE
security.provider.6=org.bouncycastle.jce.provider.BouncyCastleProvider
security.provider.7=sun.security.jgss.SunProvider
security.provider.8=com.sun.security.sasl.Provider
security.provider.9=org.jcp.xml.dsig.internal.dom.XMLDSigRI
security.provider.10=sun.security.smartcardio.SunPCSC
# the NSS security provider was not enabled for this build; it can be enabled
# if NSS (libnss3) is available on the machine. The nss.cfg file may need
# editing to reflect the location of the NSS installation.
security.provider.11=sun.security.pkcs11.SunPKCS11 
```

Reply:
Is there a reason it wouldn't work to specify `-Djava.security.properties=/path/to/overrides` instead of modifying the systemwide file?

Reply:
Oh, is it because it's a pain to control how Gradle calls Java?

Reply:
I didn't know there was such an option. I believe I just need to change:

```
org.gradle.jvmargs=-Xms256m -Xmx1024m
```

to

```
org.gradle.jvmargs=-Xms256m -Xmx1024m -Djava.security.properties=/path/to/overrides
```

Thanks for the heads up.